Adding verifiedPublisher to Application and ServicePrincipal (#9827)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/verifiedpublisher.md

---
title: "verifiedPublisher resource type"

> description: "Verified publisher of the application."

Reviewer: ```suggestion
description: "Represents the verified publisher of the application."
```

localization_priority: Normal
doc_type: resourcePageType
ms.prod: "microsoft-identity-platform"
author: "jesakowi"
---

# verifiedPublisher resource type

Namespace: microsoft.graph

> Verified publisher of the [application](application.md). For more information, see [Publisher Verification Overview](https://docs.microsoft.com/azure/active-directory/develop/publisher-verification-overview). All properties are set as a result of [Set Verified Publisher](../api/application-setverifiedpublisher.md) and can only be removed with [Unset Verified Publisher](../api/application-unsetverifiedpublisher.md).

Reviewer: ```suggestion
Represents the verified publisher of the [application](application.md). For more information, see [Publisher Verification Overview](https://docs.microsoft.com/azure/active-directory/develop/publisher-verification-overview). Verified publishers are set using [Set Verified Publisher](../api/application-setverifiedpublisher.md) and can only be removed using [Unset Verified Publisher](../api/application-unsetverifiedpublisher.md).
```
